index.js

```diff
@@ -38,16 +39,66 @@ module.exports.pitch = function(request) {
 			compilation.cache = compilation.cache[subCache];
 		}
 	});
+
+	// If the user doesn't include the magic `__assets__` anywhere, there is no
+	// need to produce an asset file.
+	var includeAssets = false
+
+	// I don't know how to register this in webpack to get a hashed filename.
+	// This is my temporary hack to use while I try to find a good solution.
+	var BAD_HASH = Math.random().toString(36).substring(2, 10)

```

Comment:
@sokra Do you know how this can be fixed? I'd really like to get this PR production ready and merged, but I'm lost.

It functions, it just uses two hacks that I think should be fixed before merging.